.github/workflows/prettier.yml

```diff
@@ -0,0 +1,31 @@
+name: Prettier check
+
+on:
+  push:
+    branches-ignore:
+      - dev
+      - main
+  pull_request:
+    branches:
+      - dev
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: "14"
+
+      - name: Install dependencies
+        run: npm install
+        working-directory: ./new
+
+      - name: Run Prettier
+        run: npx prettier --config ../.prettierrc --check .
+        working-directory: ./new
```

Comment:
When I try running this command manually, I got this output:

```
[warn] package.json
[warn] public/index.html
[warn] public/manifest.json
[warn] README.md
[warn] src/App.scss
[warn] src/App.test.tsx
[warn] src/App.tsx
[warn] src/components/button/button.scss
[warn] src/components/button/button.tsx
[warn] src/components/nav/nav.scss
[warn] src/components/nav/nav.tsx
[warn] src/constants/urls.tsx
[warn] src/index.css
[warn] src/index.tsx
[warn] src/react-app-env.d.ts
[warn] src/reportWebVitals.ts
[warn] src/sections/about/about.tsx
[warn] src/sections/contact/contact.tsx
[warn] src/sections/home/home.tsx
[warn] src/sections/portfolio/portfolio.tsx
[warn] src/setupTests.ts
[warn] tsconfig.json
[warn] Code style issues found in 22 files. Run Prettier with --write to fix.
```

When I try running with `--write` option, it overwrites all files, but git shows that there is no content change. Is it possible that it is changing the endline character? (LF - CRLF, Im not too sure abt this). Will we be able to ignore line changes in prettier config?

Reply:
Yep, endline character is likely the issue, as when I specify prettier config to CRLF on my end, the same thing you described happens too.

[reference](https://prettier.io/docs/en/options.html#end-of-line)

Please verify it's fixed after standardising the EOL config, thanks!